sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -5725,6 +5744,46 @@
       "addons": [],
       "inputType": "hidden",
       "id": "ejidpjh"
+    },
+    {
+      "input": true,
+      "tableView": true,
+      "key": "studyPeriodMinDays",
+      "label": "Study period minimum days",
+      "protected": false,
+      "unique": false,
+      "persistent": true,
+      "type": "hidden",
+      "tags": [],
+      "conditional": {
+        "show": "",
+        "when": null,
+        "eq": ""
+      },
+      "properties": {},
+      "lockKey": true,
+      "customDefaultValue": "",
+      "calculateValue": "value = data.offeringIntensity === 'Full Time' ? 84 : 42;"

```

Comment:
Since we have the values on the form now we can use `{{data.studyPeriodMinDays}}` and `{{data.studyPeriodMaxDays}}` here, right?